src/Nethermind/Nethermind.HealthChecks.Test/HealthChecksWebhookInfoTests.cs

```diff
@@ -0,0 +1,31 @@
+using System.Net;
+using System;
+using System.Collections.Generic;
+using Nethermind.JsonRpc;
+using NSubstitute;
+using NUnit.Framework;
+using Nethermind.HealthChecks;
+
+namespace Nethermind.HealthChecks.Test
+{
+    public class HealthChecksWebhookInfoTests
+    {
+        [TestCase(new byte[]{127, 64, 34, 0}, "description", "nodeName")]
+        [TestCase(new byte[]{0, 0, 0, 0}, "", "")]
+        [TestCase(new byte[]{127, 64, 34, 0}, "D_E S-C.R0IPTION", "n.o'd_e-N7a/m]e")]
+        public void HealthChecksWebhookInfo_returns_expected_results(byte[] ip_bytes, string description, string nodeName)
+        {
+            IPAddress ip = new IPAddress(ip_bytes);
+            HealthChecksWebhookInfo healthChecksWebhookInfo = new HealthChecksWebhookInfo(description, ip, nodeName);
+            string hostname = Dns.GetHostName();
+
+            string expected;
+            expected = "`" + description + "`" + Environment.NewLine
```

Comment:
Instead of concatenate strings for expected value I recommend write the full string that you're expecting and mock Ip Resolver